utils/dependencies.bzl

```diff
@@ -29,7 +31,34 @@ cc_library(
     visibility = ["//visibility:public"],
 )
     """)
+    
+#   _maybe(
+#     http_archive,
+#     name = "torch_api",
+#     urls = ["https://download.pytorch.org/libtorch/nightly/cpu/libtorch-shared-with-deps-latest.zip"],
+#     build_file_content = """
+# cc_library(
+#     name = "lib",
+#     srcs = glob(["libtorch/lib/*.*"]),
+#     hdrs = glob(["libtorch/include/**/*.h", "libtorch/include/*.h"]),
+#     visibility = ["//visibility:public"],
+# )
+#     """)
 
+  _maybe(
+    native.new_local_repository,
+    name = "torchcpp",
+    path = "./bark_ml/python_wrapper/venv/lib/python3.7/site-packages/",
```

Comment:
is this path available everywhere?

Reply:
If the user uses the provided install.sh script which creates the virtual env at the specified location then probably yes. I have tested on ubuntu but i assume the script creates the virtual env at the specified location for other unix based platform if supported.

Reply:
okay, this just makes it even more dependent on the python version

Reply:
There is a alternate, to create a  web based repository target and download the official C++ torch libs. I tried that but the libs are developed using a different c/pyhton version causing conflicts.

Reply:
It is the same kind of definition we have in BARK. So this is fine for me.